client/lib/site/utils.js

```diff
@@ -106,12 +103,15 @@ export function hasCustomDomain( site ) {
 }
 
 export function isModuleActive( site, moduleId ) {
-	return site.options.active_modules && site.options.active_modules.indexOf( moduleId ) > -1;
+	return (
+		site.options &&
+		site.options.active_modules &&
+		site.options.active_modules.indexOf( moduleId ) > -1
```

Comment:
Optional chaining comes really handy in simplifying those:

```suggestion
		site.options?.active_modules?.indexOf( moduleId ) > -1
```

Reply:
Oh, cool. We can use it in JS as well. I thought we couldn't because we aren't using in a lot of JS files, but the reason probably is because these files are older and not updating in a long time, not because the feature isn't supported. 😄 

I will update them accordingly! Thanks!

Reply:
Done!